doc/developer/mesh.md

```diff
@@ -109,6 +109,14 @@ During manipulation, if two wedges become the same (i.e. for the example on the
 To merge wedges an explicit call to TopologicalMesh::mergeEqualWedges is needed.
 When a wedge is not referenced anymore (because referencing halfedges have been deleted), it is marked for deletation. When one call Ra::Core::Geometry::TopologicalMesh::garbage_collection(), the marked wedges are eventually deleted and halfedges index are updated accordingly.
 
+During convertion from Core::Geometry::*Mesh to Core::Geometry::TopologicalMesh, non manifold mesh faces are not added but given to a optionally provided user defined functor (\see Ra::Core::Geometry::TopologicalMesh::TopologicalMesh).
+Some meshes have multiple times the **same** vertices with **different** values for attributes inside a single face.
+This kind of situation can either raise degerated faces, (in case of triangle mesh) or valid faces for some cases for polymesh (\todo will be supported when TopologicalMesh will support PolyMesh).
+In the latter case, the face is cleaned from the duplicated position and added to the TopologicalMesh, using the first encountered wedge value.
```

Comment:
Why the first. Is there a way, for polymesh, to specify which wedge to take (e.g. The one with the maximum cotangent weight) ?

Reply:
arbitrarly ... their is no cotan weight here, the position are the same .... 
the other solution is to drop them to the functor for non manifold faces.

Reply:
see figure (here in the comment, or [updated in the doc](https://github.com/STORM-IRIT/Radium-Engine/blob/f7e57ca4e7c5ca01cc33e61433a12678f1aa72b6/doc/images/wedge-degen.svg))